governance.md

```diff
@@ -0,0 +1,105 @@
+# Principles
+
+The Confidential Containers is an open source organization and community
+that adheres to the following principles:
+
+* Open - Confidential Containers is an open source project. We welcome all [contributions](https://github.com/confidential-containers/community/blob/main/CONTRIBUTING.md)
+* Respectful and welcoming - See our [Code of Conduct](https://github.com/confidential-containers/community/blob/main/CODE_OF_CONDUCT.md)
+* Transparent - All work is done in public, either from our GitHub repositories or from public meetings.
```

Comment:
Agreed. A re-wording of this would be beneficial. We obviously would want to keep things like CVE's, etc, private until appropriate actions are taken/tested, approved, and released. That being said, I do like highlighting that we are as transparent as possible. Would it be beneficial to add in a small snippet about licensing here, as well?

Reply:
Absolutely we need commercial friendly license models. But should it be defined by the TOC team?

Reply:
The licensing is already specified in the `CONTRIBUTING` document, but I'm adding the fact that the project is licensed under the Apache 2 license in the `Principles` section.

Reply:
I'll also reword the `Transparent` part.

Reply:
Looks good. I change that part with your input, @magowan